torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -675,6 +677,17 @@ ProcessGroupNCCL::ProcessGroupNCCL(
             << "\nUSE_HIGH_PRIORITY_STREAM: "
             << options_->is_high_priority_stream;
 
+  RECORD_PARAM_COMMS(
```

Comment:
Reason for adding this? (since initialization is not a collective)

Reply:
The network.ai team requested this feature. We can collect when a network collective is created during initialization. cc: Pavani.